nms/app/fbcnms-projects/magmalte/app/views/network/NetworkRanConfig.js

```diff
@@ -67,99 +92,116 @@ type Props = {
 
 export default function NetworkRan(props: Props) {
   const classes = useStyles();
-  const typographyProps = {
-    primaryTypographyProps: {
-      variant: 'caption',
-      className: classes.itemTitle,
-    },
-    secondaryTypographyProps: {
-      variant: 'h6',
-      className: classes.itemValue,
-    },
-  };
   const [open, setOpen] = React.useState(true);
 
+  const bandwidth: KPIRows[] = [
+    [
+      {
+        category: 'Bandwidth',
+        value: props.lteRanConfigs?.bandwidth_mhz || '-',
+      },
+    ],
+  ];
+
+  const tdd: KPIRows[] = [
+    [
+      {
+        category: 'EARFCNDL',
+        value: props.lteRanConfigs?.tdd_config?.earfcndl || '-',
+      },
+    ],
+    [
+      {
+        category: 'Special Subframe Pattern',
+        value: props.lteRanConfigs.tdd_config?.special_subframe_pattern || '-',
+      },
+    ],
+    [
+      {
+        category: 'Subframe Assignment',
+        value: props.lteRanConfigs?.tdd_config?.subframe_assignment || '-',
+      },
+    ],
+  ];
+
+  const fdd: KPIRows[] = [
+    [
+      {
+        category: 'EARFCNDL',
+        value: props.lteRanConfigs?.fdd_config?.earfcndl || '-',
+      },
+    ],
+    [
+      {
+        category: 'EARFCNUL',
+        value: props.lteRanConfigs?.fdd_config?.earfcnul || '-',
+      },
+    ],
+  ];
+
   return (
-    <List component={Paper} data-testid="ran">
-      <ListItem>
-        <ListItemText
-          primary={'Bandwidth'}
-          secondary={props.lteRanConfigs?.bandwidth_mhz}
-          {...typographyProps}
-        />
-      </ListItem>
+    <Grid item xs={12}>
+      <KPIGrid data={bandwidth} />
```

Comment:
Why are we using KpiGrid for bandwidth instead of List ? This is also the place where the test is failing. I thought we wanted to use KpiGrid for showing KPI values alone.

Reply:
KPI Grid has evolved into a bit of a catch all. From the designs their is not difference between the way in which a KPI is styled and the way a List is done. Semantically List makes more sense, but structurally and design-wise they are the same. 

That said, there is the issue of Collapsable lists which currently can not exist within KPI Grid as it is currently structured so there are some odd examples like here, where I use KPI Grid to handle the more default looking sections of the list, and then manually code out the collapsable portion followed by another KPI Grid. I personally think that we should continue to refine KPI Grid so that it can incorporate collapsible lists. As it continues to mature it could be our catch-all for most of the content areas with the exception being tables really.

I'm also open to renaming the component so it's more indicative of what it actually can do, rather than strictly keeping the `KPI` nomenclature. Perhaps something like `DataGrid` would be a more generalized name for it?

Reply:
If we make it a catch all, as you mentioned let us rename it to DataGrid, that makes more sense. Also perhaps we should merge KPIGrid and KPITray if possible if they do similar things. Just curious, why are we using it here for bandwidth field specifically, what is the styling being added here?

Reply:
I'm totally down for renaming it. I believe KPI Grid is just more mature than KPI Tray at this point. It incorporates the ability to have multiple KPI's per row, as well as entirely new rows, allowing for one to make a list like display. We should be able to convert all `KPI Trays` over to `KPI Grid` and retire that component.

Should we save these major KPI Grid revisions for a new PR or bundle them with this? I'll have to convert old KPI Trays over, and update anywhere we use a `Collapse` list to instead use our new `DataGrid` component -- not to mention regression testing.